commands/exportcommands.py

```
@@ -160,17 +170,40 @@ def _export_hooks(self, staging_dir):
 
                     # Ensure scripts are executable
                     if file.endswith(('.sh', '.py')):
-                        os.chmod(file_path, 0o755)
+                        try:
+                            os.chmod(file_path, 0o755)
+                        except:
+                            pass  # Windows may not support chmod
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

(SIM105)